packages/framework/src/Resources/config/packages_registry.yaml

```diff
@@ -1,20 +1,20 @@
-        - { name: 'brand-feed-luigis-box', path: '%shopsys.framework.root_dir%/../brand-feed-luigis-box', namespace: 'Shopsys\BrandFeed\LuigisBoxBundle', app_namespace: 'App\BrandFeed\LuigisBox' }
-        - { name: 'category-feed-luigis-box', path: '%shopsys.framework.root_dir%/../category-feed-luigis-box', namespace: 'Shopsys\CategoryFeed\LuigisBoxBundle', app_namespace: 'App\CategoryFeed\LuigisBox' }
-        - { name: 'coding-standards', path: '%shopsys.framework.root_dir%/../coding-standards', namespace: 'Shopsys\CodingStandards', app_namespace: 'App\CodingStandards' }
-        - { name: 'form-types-bundle', path: '%shopsys.framework.root_dir%/../form-types-bundle', namespace: 'Shopsys\FormTypesBundle', app_namespace: 'App\FormTypes' }
-        - { name: 'google-cloud-bundle', path: '%shopsys.framework.root_dir%/../google-cloud-bundle', namespace: 'Shopsys\GoogleCloudBundle', app_namespace: 'App\GoogleCloud' }
-        - { name: 'http-smoke-testing', path: '%shopsys.framework.root_dir%/../http-smoke-testing', namespace: 'Shopsys\HttpSmokeTesting', app_namespace: 'App\HttpSmokeTesting' }
-        - { name: 'luigis-box', path: '%shopsys.framework.root_dir%/../luigis-box', namespace: 'Shopsys\LuigisBoxBundle', app_namespace: 'App\LuigisBox' }
-        - { name: 'migrations', path: '%shopsys.framework.root_dir%/../migrations', namespace: 'Shopsys\MigrationBundle', app_namespace: 'App\Migration' }
-        - { name: 'plugin-interface', path: '%shopsys.framework.root_dir%/../plugin-interface', namespace: 'Shopsys\Plugin', app_namespace: 'App\Plugin' }
-        - { name: 'product-feed-google', path: '%shopsys.framework.root_dir%/../product-feed-google', namespace: 'Shopsys\ProductFeed\GoogleBundle', app_namespace: 'App\ProductFeed\Google' }
-        - { name: 'product-feed-heureka', path: '%shopsys.framework.root_dir%/../product-feed-heureka', namespace: 'Shopsys\ProductFeed\HeurekaBundle', app_namespace: 'App\ProductFeed\Heureka' }
-        - { name: 'product-feed-heureka-delivery', path: '%shopsys.framework.root_dir%/../product-feed-heureka-delivery', namespace: 'Shopsys\ProductFeed\HeurekaDeliveryBundle', app_namespace: 'App\ProductFeed\HeurekaDelivery' }
-        - { name: 'product-feed-luigis-box', path: '%shopsys.framework.root_dir%/../product-feed-heureka-delivery', namespace: 'Shopsys\ProductFeed\LuigisBoxBundle', app_namespace: 'App\ProductFeed\LuigisBox' }
-        - { name: 'product-feed-zbozi', path: '%shopsys.framework.root_dir%/../product-feed-zbozi', namespace: 'Shopsys\ProductFeed\ZboziBundle', app_namespace: 'App\ProductFeed\Zbozi' }
-        - { name: 's3-bridge', path: '%shopsys.framework.root_dir%/../s3-bridge', namespace: 'Shopsys\S3Bridge', app_namespace: 'App\S3Bridge' }
+        framework: { path: '%shopsys.framework.root_dir%', namespace: 'Shopsys\FrameworkBundle', app_namespace: 'App' }
```

Comment:
I believe it would be nice to mention this file in `AbstractShopsysReleaseWorker` and `monorepo_functions.sh` (there are comments that remind us about the places where all the packages are listed)